src/main/java/teammates/ui/controller/AdminInstructorAccountAddAction.java

```diff
@@ -294,13 +315,18 @@ private String generateNextDemoCourseId(String instructorEmailOrProposedCourseId
         }
     }
 
+    /**
+     * This function trims the course ID if it is too long.
+     * @param demoCourseId
+     * @param maximumIdLength length to trim
+     * @return trimmed Course ID
+     */
     private String trimCourseIdToMaximumLengthIfNecessary(String demoCourseId, final int maximumIdLength) {
         final int courseIdLength = demoCourseId.length();
         if (courseIdLength <= maximumIdLength) {
             return demoCourseId;
         } else {
-            return demoCourseId.substring(courseIdLength
-                    - maximumIdLength);
+            return demoCourseId.substring(courseIdLength - maximumIdLength);
         }
     }
```

Comment:
I think it is still worth putting it there on the off chance that someone else might need it one day. It can be called `truncateHead()`? Can you create a separate issue for it, for contributors?